library/src/main/java/org/xmtp/android/library/Contacts.kt

```diff
@@ -45,70 +54,103 @@ class ConsentList(val client: Client) {
         client.privateKeyBundleV1.identityKey.publicKey.secp256K1Uncompressed.bytes
     private val privateKey = client.privateKeyBundleV1.identityKey.secp256K1.bytes
 
-    private val identifier: String = uniffi.xmtpv3.generatePrivatePreferencesTopicIdentifier(
-        privateKey.toByteArray()
-    )
+    private val identifier: String =
+        uniffi.xmtpv3.generatePrivatePreferencesTopicIdentifier(
+            privateKey.toByteArray(),
+        )
 
     @OptIn(ExperimentalUnsignedTypes::class)
     suspend fun load(): ConsentList {
-        val envelopes = client.apiClient.envelopes(
-            Topic.preferenceList(identifier).description,
-            Pagination(direction = MessageApiOuterClass.SortDirection.SORT_DIRECTION_ASCENDING)
-        )
+        val envelopes =
+            client.apiClient.envelopes(
+                Topic.preferenceList(identifier).description,
+                Pagination(direction = MessageApiOuterClass.SortDirection.SORT_DIRECTION_ASCENDING),
+            )
         val consentList = ConsentList(client)
         val preferences: MutableList<PrivatePreferencesAction> = mutableListOf()
         for (envelope in envelopes) {
-            val payload = uniffi.xmtpv3.userPreferencesDecrypt(
-                publicKey.toByteArray(),
-                privateKey.toByteArray(),
-                envelope.message.toByteArray()
-            )
+            val payload =
+                uniffi.xmtpv3.userPreferencesDecrypt(
+                    publicKey.toByteArray(),
+                    privateKey.toByteArray(),
+                    envelope.message.toByteArray(),
+                )
 
             preferences.add(
                 PrivatePreferencesAction.parseFrom(
-                    payload.toUByteArray().toByteArray()
-                )
+                    payload.toUByteArray().toByteArray(),
+                ),
             )
         }
 
         preferences.iterator().forEach { preference ->
-            preference.allow?.walletAddressesList?.forEach { address ->
+            preference.allowDm?.walletAddressesList?.forEach { address ->
```

Comment:
I probably should have commented this on the proto PR. But you're denying an address in this case so it should honestly be `allowAddress` or `denyAddress` since we also use this functionality inside groups to change the way messages appear in a group context for a denied address. But this doesn't get surfaced to the user so probably not a huge deal.

Reply:
Please see https://github.com/xmtp/proto/pull/149 and consider ✅  that one first.  And I'll use update this PR to leverage that one.